app/views/schedules/_task_list.html.erb

```diff
@@ -1,29 +1,54 @@
-<ul>
-  <% if schedule.root_task.nil? %>
-      <li>No root task picked...</li>
-  <% else %>
-      <% schedule.root_task.self_and_descendants.each do |task| %>
-      <li>
-        <span class="label label-primary">
-        <%= link_to ' ',
-                    task_run_path(task),
-                    method: :post,
-                    class: 'text-danger glyphicon glyphicon-off',
-                    alt: "Run Task ##{task.id} Isolated",
-                    title: "Run Task ##{task.id} Isolated" %>
-          <% if task.id == schedule.root_task.id %>
-              <span class="glyphicon glyphicon-flash"></span>
-          <% end %>
-          Task #<%= task.id %>:
-        </span>
-        <br />
-        <span class="label label-success"><%= task.type_name_display %></span>
-        <% if !task.equipment.nil? %>
-            <%= render partial: 'schedules/equipment_task_details', locals: {task: task} %>
-        <% elsif !task.thermostat.nil? %>
-            <%= render partial: 'schedules/thermostat_task_details', locals: {task: task} %>
-        <% end %>
-      </li>
-      <% end %>
-  <% end %>
-</ul>
\ No newline at end of file
+
+
+<html>
+<head>
+  <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
```

Comment:
Before you're done, you should pull the Javascript portions into a file in the app/assets/javascript folder that can get called from here via the assets pipeline.